ddl/table.go

```diff
@@ -309,3 +316,15 @@ func updateTableInfo(t *meta.Meta, job *model.Job, tblInfo *model.TableInfo, ori
 
 	return ver, t.UpdateTable(job.SchemaID, tblInfo)
 }
+
+// cleanAllocatorKey cleans up auto ID key.
+func cleanAllocatorKey(t *meta.Meta, tblInfo *model.TableInfo) error {
```

Comment:
Can we remove the `delAutoID` argument in `meta.DropTable` and do it in this method?

Reply:
+1